docs/gettsim_objects/input_variables.md

```diff
@@ -572,3 +572,15 @@ Type: int
 Number of own children below the age of 25 (incl. not in hh)
 
 Type: int
+
+## `budget_erzgeld`
```

Comment:
```suggestion
## `budgetsatz_erzieh`
```
Makes searching slightly easier, at least. And `budget` by itself is too ambiguous (if not familiar with the context).

Will change before being released, anyhow.